pkg/services/alerting/notifiers/line.go

```diff
@@ -75,6 +75,11 @@ func (this *LineNotifier) createAlert(evalContext *alerting.EvalContext) error {
 	body := fmt.Sprintf("%s - %s\n%s", evalContext.Rule.Name, ruleUrl, evalContext.Rule.Message)
 	form.Add("message", body)
 
+        if evalContext.ImagePublicUrl != "" {
+        form.Add("imageThumbnail", evalContext.ImagePublicUrl)
```

Comment:
@huydx sure, i can delete imageThumbnail. close this PR and create new one
owh and i got 1 more bugs that i need to look for. grafana were sending OK alert response to LINE but it didn't get to LINE chat room eventually for OK response. i'll look for it when i got free time
for thumbnail image, perhaps better look at other needs first, if other alert (telegram, whatsapp, etc) also need thumbnail support. then we can discuss for it